### PR TITLE
week1 project implementation

### PR DESCRIPTION
**Pull request by bhavikadalwadi**

### CODE REVIEW

File: opensearch/bbuy_queries.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+  "mappings": {
+    "properties": {
+      "user": {"type": "keyword"},
+      "sku": {"type": "long"},
+      "category": {"type": "keyword"},
+      "query":{
+        "type": "text",
+        "analyzer": "english",
+        "fields": {
+          "keyword": {
```
Comment (Harshit-Taiger) [inline opensearch/bbuy_queries.json:14]: can consider using stemming here

File: week1/search.py [hunk 2]
```diff
@@ -90,11 +130,82 @@ def create_query(user_query, filters, sort="_score", sortDir="desc"):
...
+                                    "fields": ["name^100", "shortDescription^50", "longDescription^10", "department"],
+                                    "query": user_query
+                                }
+                            }
+                        ],
+                        "filter": filters
+                    }
+                },
+                "boost_mode": "multiply",
+                "score_mode": "avg",
```
Comment (Harshit-Taiger) [inline week1/search.py:150]: looks good

File: opensearch/bbuy_queries.json (hunk 1 above)
Comment (bhavikadalwadi) [reply opensearch/bbuy_queries.json:14]: I thought "english" analyzer will also perform stemming too ?

File: week1/search.py (hunk 2 above)
Comment (bhavikadalwadi) [reply week1/search.py:150]: thank you!

File: opensearch/bbuy_queries.json (hunk 1 above)
Comment (Harshit-Taiger) [reply opensearch/bbuy_queries.json:14]: Ohh yes right, english analyser itself will provide stemming also, https://qbox.io/blog/elasticsearch-english-analyzer-customize , There are various stemmers so should be ok.

**Pull request closed without merging**